_posts/2017-06-06-inklink.html

```diff
@@ -0,0 +1,65 @@
+
+<p>Picture this: you are at home, reading your favourite financial newspaper with your morning coffee, spreading pages over your kitchen table. It is nigh impossible for you to take those pages along for your commute. What if you could snap the main image with your smartphone and have the online equivalent to browse on the go? This is one of the aims we set for InkLink.</p>
+
+<h2>Augmented Reality (AR) in newspapers</h2>
+<p>Back in 2011, long before Pokémon Go, there was a resurgence in AR applications, especially with advertisers. Most newspapers, even the FT, urged their readers to scan their adverts and see them come to life. If this innovative way of consuming content was fun for a short while, the novelty wore off rapidly. Seen as more of a gimmick the solution proved impractical both for readers -- who had to hold their phone or tablet in place after having downloaded a new application -- and editors, who would have to indicate scannable content by adding markers (which, aesthetically, can be as bad as QR codes) and sourcing content. It transpired this was a lot of preparation for a rather small pay-off.</p>
+
+<p>With InkLink, we focused on the step that comes before AR, recognising a pre-specified location in the paper using Image Recognition (AR only works in the presence of patterns, or images, to recognise). Our premise is the following: can we help our readers digitally bookmark article for future reference by a simple scan of their morning paper?</p>
+
+<h2>What InkLink can do</h2>
+<p>Our research into InkLink has shown that the main difficulty, beyond technical constraints, is to surface article data across channels. Unfortunately, the text printed in the FT, isn’t currently matched in any way to online content. However, with manually curated content, our tool can recognise pictures from the paper and deliver up-to-date, additional content; complete with comments and related articles. It could also reveal interactive graphics or videos, not as an augmented layer, but straight out of ft.com, as a companion to the printed version.</p>
+
+<o>InkLink can recognise a variety of images as long as they’re the focal point of the capture. It could even recognise the daily crossword grid (blank or partially filled in).</p>
+
+<p>To achieve this result, we explored number of third-party services that focused on AR.</p>
+
+<h2>Technical details</h2>
+<p>After looking into Image Retrieval (the technique employed by Google reverse image search), it became apparent that we didn’t need to identify elements within an image, just recognise it. That’s why we simplified our approach and are now focusing on Image Recognition. To get it working, we compared 3 different services: Blippar, Aurasma, and CraftAR by Catchoom.</p>
```

Comment:
Missing links to the 3 services we tried:
[Blippar](https://blippar.com/en/)
[Aurasma](https://www.aurasma.com/)
[Catchoom](https://catchoom.com/solutions/image-recognition/)